offchain/danaswap-api/src/DanaSwap/Api.purs

```diff
@@ -88,6 +94,31 @@ instance EncodeAeson Protocol where
       }
 
 type PoolId = TokenName
+type AssetClass = CurrencySymbol /\ TokenName
+
+newtype PoolDatum =
+  PoolDatum
+    { ac1 :: AssetClass
+    , ac2 :: AssetClass
+    , bal1 :: BigInt
+    , bal2 :: BigInt
+    , adminBal1 :: BigInt
+    , adminBal2 :: BigInt
+    , liquidity :: BigInt
```

Comment:
Could we rename this to `lt`? It's arguably shorter and more descriptive (at least for me, as I've been using `lt` for liquidity tokens in my maths)

Reply:
I really like issued liquidity is that name good with you?